libraries/tools/kotlin-stdlib-gen/src/templates/Aggregates.kt

```diff
@@ -840,6 +840,63 @@ object Aggregates : TemplateGroupBase() {
         }
     }
 
+    val f_reduceOrNull = fn("reduceOrNull(operation: (acc: T, T) -> T)") {
+        include(ArraysOfPrimitives, ArraysOfUnsigned, CharSequences)
+    } builder {
+        inline()
+        specialFor(ArraysOfUnsigned) { inlineOnly() }
+
+        doc { "Accumulates value starting with the first ${f.element} and applying [operation] from left to right to current accumulator value and each ${f.element}. Returns null if the first ${f.element} does not exist." }
```

Comment:
Use multiline doc string here and describe return value with the `@return` kdoc tag, e.g.
```
@return the resulting accumulator value or `null` if ...
```

`"... if the first element does not exists"` sounds strange, as it might be that the first element does not exist, but the second does.  Better to say just `"if the ${f.collection} is empty."`

Reply:
Good point, changed!